lib/index.js

```diff
@@ -105,29 +132,67 @@ class Suite {
     throwIfNoNativesSyntax();
     const results = new Array(this.#benchmarks.length);
 
-    // This is required to avoid variance on first benchmark run
-    for (let i = 0; i < this.#benchmarks.length; ++i) {
-      const benchmark = this.#benchmarks[i];
-      debugBench(`Warmup ${ benchmark.name } with minTime=${ benchmark.minTime }, maxTime=${ benchmark.maxTime }`);
-      const initialIteration = await getInitialIterations(benchmark);
-      await runWarmup(benchmark, initialIteration, { minTime: 0.005, maxTime: 0.05 });
+    // It doesn't make sense to warmup a fresh new instance of Worker.
+    // TODO: support warmup directly in the Worker.
+    if (!this.#useWorkers) {
+      // This is required to avoid variance on first benchmark run
```

Comment:
Initially I added this warmup to try to trigger v8 optimizations, but since we use %DoNotOptimize, does this warmup makes any difference?

Reply:
I think it does as plugins are manageable. We use the never optimize plugin by default, but users could re-define that.